### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by dshanske**
Returns a jf2 h-card in the token return on verification so the client can display name and user avatar on an application

### CODE REVIEW

**Review by pfefferle — APPROVED** [review]

File: includes/functions.php [hunk 1]
```diff
@@ -410,3 +410,29 @@ function indieauth_get_me() {
...
+	if ( is_numeric( $user ) ) {
+		$user = get_user_by( 'ID', $user );
+	}
+	if ( ! $user instanceof WP_User ) {
+		return array();
+	}
+	$return = array(
+		'type'  => 'card',
+		'name'  => $user->display_name,
+		'url'   => empty( $user->user_url ) ? get_author_posts_url( $user->ID ) : $user->user_url,
```
Comment (pfefferle) [inline includes/functions.php]: shouldn't we always use the `author_posts_url`? Because the `user_url` might be a completely different domain.

**Comment by pfefferle** [comment]
Is it required to send the correct `application/jf2feed+json` content-type?

https://www.w3.org/TR/jf2/#jf2feed_discovery

File: includes/functions.php (hunk 1 above)
Comment (dshanske) [reply includes/functions.php]: Remember, this is a profile return. Not the 'me' return which is part of the client_id process, and shouldn't change. This is what the person wants to share as their site.

**Comment by dshanske** [comment]
@pfefferle JSON needs to be the return for an Indieauth response.

**Comment by pfefferle** [comment]
A, have to read more carefully... this is part of the response, not an extra endpoint protected by the access_token.

**Comment by dshanske** [comment]
Theoretically, if someone wanted more than that, they could ask for it with a new scope, but this is just the basic info. It is now live in Quill, if you want to try logging into that after the update goes live.

**Pull request merged**